server/server.go

```diff
@@ -143,7 +143,7 @@ func Serve(efs EmbedFS) {
 
 	denoStdVersion, err = getDenoStdVersion()
 	if err != nil {
-		log.Fatalf("getDenoStdVersion: %v", err)
+		log.Warnf("getDenoStdVersion: %v", err)
```

Comment:
If you are not comfortable with my suggestion, please add a TODO / FIXME here to look at it later.

In `build.go`, theree's a place `denoStdVersioin` is used (linie 487).

There, it needs to check if `denoStdVersioin` is set, and if not build a string without it.

Example, this is what it would look like with it set: https://deno.land/std@0.139.0/archive/tar.ts
This is what it would look like without it set: https://deno.land/std/archive/tar.ts

The URL without it still works without the version with a redirect, so this is a valid change, but really only with the string update because that `@` in the URL is invalid without a version there.